source/randLIB.c

```diff
@@ -95,14 +95,13 @@ void randLIB_seed_random(void)
      * multiple times,or in case someone has called randLIB_add_seed before
      * this. We don't want to potentially lose entropy.
      */
-
-    srand(time(0));
+    randLIB_add_seed(time(NULL));
     /* Spell out expressions so we get known ordering of 4 seed calls */
-    uint64_t s = (uint64_t) rand() << 32;
-    state[0] ^= (s | rand());
+    uint64_t s = (uint64_t) splitmix64(&state[0]) << 32;
```

Comment:
I'm still wary about the whole "attempt to seed from time" concept - see other comment.

Regardless, in case this is retained: the whole picture has ended up a bit confused here, with the manual state manipulation bracketed by two add seed calls.

All would want is a single `randLIB_add_seed(time(NULL))`. You don't need to mess further. LIke this. All of 99-109 are pointless.

Also, all the comments in this function are now out-of-date after these 2 PRs. You can't change the code and leave comments referring to old code/algorithms.